beamerouterthemeusyd-logobar.sty

```diff
@@ -0,0 +1,57 @@
+\mode<presentation>
+
+\beamer@compresstrue
+
+% This is the padding between text elements and the edges of the colorboxes
+\newlength{\theme@hpadding}
+\setlength{\theme@hpadding}{1.5em}
+
+% This is the whitespace on the top and bottom of the slides.
+\newlength{\theme@vpadding}
+\setlength{\theme@vpadding}{0.5em}
+
+\defbeamertemplate*{frametitle}{usyd logo}[1][]
+{%
+  \vskip3em
```

Comment:
I think this value is the top margin you are referring to being too large. Changing this line to `\vskip2em` brings the slide title closer to the logobar.

Reply:
I think this might be too large also, but I was actually referring to the margin north of the logobar

Reply:
I have made modifications compressing all the top and bottom components. Is that more appropriate?
<img width="808" alt="screen shot 2018-06-18 at 18 59 37" src="https://user-images.githubusercontent.com/10113564/41527244-d7d453a6-7329-11e8-974b-011dc88130a0.png">